iosApp/Modules/Screens/Library/UI/FilterView.swift

```diff
@@ -0,0 +1,38 @@
+//import SwiftUI
+//import shared
+//import AudioPlayer
+//
+//struct FilterSortMenuView: View {
```

Comment:
Can we remove the code we are using here?

Reply:
bah, yep, that was the chatgpt experiment.  it did ok 😐